.github/PULL_REQUEST_TEMPLATE

```diff
@@ -13,23 +13,25 @@ Package(s) Affected
 Security Update?
 ----------------
 
-<!-- If this topic is part of a security update, please uncomment "Yes,"
-     and mark with the `security` label, as well as reference issue number below for priority processing. -->
+<!-- If this topic contains security update(s), please uncomment "Yes,"
+     and mark with the `security` and `priority` labels, as well as reference issue number below for prioritised processing. -->
 
 <!-- Yes - Issue Number: ISSUENUMBER -->
```

Comment:
I prefer to use [Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) feature instead.

Reply:
Got it.